### PR TITLE
Adding manual pages, also adding manual installation step

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

**Comment by domcleal** [comment]
I think you also need to copy `manuals/1.1` too, otherwise :+1:

**Comment by lzap** [comment]
Fixed, also changed 1.1 strings in there and added Rakefile.

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          puts "#{referrer.url} contains the following broken links:"
+          pages.each do |page|
+            puts "  HTTP #{page.code} #{page.url}"
+          end
+        end
+      end
+    end
+
+  rescue LoadError
+    abort 'Install anemone gem: gem install anemone'
```
Comment (domcleal) [inline Rakefile]: Please add this to the Gemfile.

File: manuals/1.2/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
+layout: manual
+title: Foreman 1.2 Manual
+---
+
+# 1. Foreman 1.2 Manual
+{% include manuals/1.2/1.2_architecture.md %}
```
Comment (domcleal) [inline manuals/1.2/index.md]: Search & replace gone wrong?  Should be 1.1_architecture.

**Comment by domcleal** [comment]
I can't get the new check_links rake task to work - any idea what I'm missing?

<pre>
$ rake check_links
iridium - - [01/May/2013:11:26:16 BST] "GET / HTTP/1.1" 200 4746
- -> /
</pre>

File: manuals/1.2/index.md (hunk 2 above)
Comment (lzap) [inline manuals/1.2/index.md]: Oh right, found one more like this one. Good catch.

**Comment by domcleal** [comment]
Thanks @lzap, merged!

**Pull request merged**